### PR TITLE
Bug 1944865 - Do not clear client-info on disabling collection

### PR DESCRIPTION
**Pull request by badboy**
This data might end up in no-follows-collection-enabled pings. That's fine.

---

Is it really that easy? Let CI decide*!

* not really though, we will of course double-check that we're not forgetting anything.

### CODE REVIEW

**Review by travis79 — APPROVED** [review]
I _think_ this looks okay. I'm a bit surprised it was this simple...

**Review by travis79 — APPROVED** [review]

**Comment by badboy** [comment]
I might have spotted another edge case, so I'm gonna write a test for that first

File: tools/dbread.rs [hunk 1]
```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env -S cargo +nightly -q -Zscript
+---cargo
+[dependencies]
+bincode = "1.0"
+serde = {version = "1.0.144", features = ["derive"]}
+bitflags = {version = "2.4.1", features = ["serde"]}
+tabwriter = "1.1.0"
+---
```
[the remark this answers is not in the document]
Comment (badboy) [reply tools/dbread.rs:8]: hell yeah! https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#script

[the remark this answers is not in the document]
Comment (badboy) [reply tools/dbread.rs:8]: and of course https://fnordig.de/2023/12/28/cargo-script-and-cgi/

File: tools/dbread.rs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                let m: Metric = bincode::deserialize(&d).unwrap();
+                writeln!(&mut tw, "{k}\t{m:?}").ok();
+            } else {
+                writeln!(&mut tw, "{k}\t{:?}..", &v[..4]).ok();
+            }
+        }
+        writeln!(&mut tw, "").ok();
+        tw.flush().unwrap();
+    }
+}
```
[the remark this answers is not in the document]
Comment (badboy) [reply tools/dbread.rs:102]: 🥖

**Review by travis79 — APPROVED** [review]
